Assets/Shapes2D/Scripts/Shape.cs

```diff
@@ -923,8 +924,8 @@ protected class ShaderProps {
                 bottomLeft = transform.InverseTransformPoint(bottomLeft);
                 topRight = transform.InverseTransformPoint(topRight);
                 bottomRight = transform.InverseTransformPoint(bottomRight);
-                size.x = Vector3.Distance(topLeft, topRight);
-                size.y = Vector3.Distance(topLeft, bottomLeft);
+                size.x = Vector2.Distance(topLeft, topRight);
+                size.y = Vector2.Distance(topLeft, bottomLeft);
```

Comment:
IIRC I changed this to Vector3 at some point but now I can't remember if it was fixing an actual problem or just a random change...I think it would be better to leave it the way it was just in case, since I don't think this is needed by your PR